### ISSUE TITLE
My Site Dashboard - Fix Quick Start snackbar issues

### ISSUE THREAD

**Issue by ashiagr** [wordpress-mobile/WordPress-Android#15296]
Parent #15125

The goal of this task is to fix pre-existing Quick Start snackbar issues: 

- [x] #15430 ⚙️  @ashiagr 
- [x] #15461 ⚙️ @ashiagr
- [x] #15462 ⚙️ @ashiagr

See https://github.com/wordpress-mobile/WordPress-Android/issues/15258#issuecomment-910489481 for more details.

**Comment by yaelirub** [comment]
Hello 👋  I was following a conversation on Slack regarding the space at the bottom of the screen and followed the links @ParaskP7 shared to this issue and to https://github.com/wordpress-mobile/WordPress-Android/issues/15430

A couple of questions:
1. Is https://github.com/wordpress-mobile/WordPress-Android/issues/15430 a duplicate of this and if so can we close it ?
2. Can we please fix this issue as part of the work on My Site? I think this is more than nice to have as we are getting reports

**Comment by ashiagr** [comment]
Hey @yaelirub 👋 

Thanks for bringing it to notice.  🙇‍♀️ 

> 1. Is #15430 a duplicate of this and if so can we close it ?

Actually, #15430 is one amongst few other (pre-existing) `Quick Start` issues included in this task. So I've kept issue `#15430` open and created separate issues for others so that we can track them better. 

> 2. Can we please fix this issue as part of the work on My Site? I think this is more than nice to have as we are getting reports

I've renamed this task since the "Nice to haves" title looks misleading.  #15430 was a bit tricky to resolve when we investigated in Phase 1 of the project, given the fact that it auto resolves when the screen is slightly scrolled. Looking at the reports, we'll prioritize it in our list of tasks for `My Site Dashboard` project.
